### ISSUE TITLE
Can we add a section on 'free environments to teach yourself programming'

### ISSUE THREAD

**Issue by Joykareko** [EbookFoundation/free-programming-books#6057]
Since the library contains free programming resources, can we also add a list of websites where you can teach yourself programming for free using their online free coding environments? Such environments are like  [_Grasshopper_](https://learn.grasshopper.app/).

**Comment by davorpa** [comment]
> Since the library contains free programming resources, can we also add a list of websites where you can teach yourself programming for free using their online free coding environments? Such environments are like [_Grasshopper_](https://learn.grasshopper.app/).

@Joykareko, maybe you are refering to our [`Programming Playgrounds` listings](https://github.com/EbookFoundation/free-programming-books#programming-playgrounds)?

**Comment by Joykareko** [comment]
@davorpa, yes. I would like to add the site to programming Playground listings. Could you please assign me this issue? I would like to work on it.

**Comment by davorpa** [comment]
> I would like to add the site to programming Playground listings. Could you please assign me this issue? I would like to work on it.

Remember [link in the request body with something like `Resolves #NNNN`](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue), where NNNN is provided by GitHub when the PR is created, :pray: thanks.
